examples/stocks/README.md

```diff
@@ -1,3 +1,12 @@
+### Stocks
+
+Stocks is an older example app of using Flutter (than flutter_gallery), but is still useful for several things:
```

Comment:
I wouldn't mention that it's older. It comes over as apologetic.

Here's what I wrote in its entry for ../README.md (PR to be submitted shortly):

```
- **Stocks** The [stocks](stocks) demo shows how one might structure
  an application with several screens.
```